### PR TITLE
Test/20200102

### PR DESCRIPTION
**Pull request by opoudjis**

### CODE REVIEW

File: lib/isodoc/ogc/html/ogc.scss [hunk 1]
```diff
@@ -86,7 +86,7 @@ p.SourceTitle
         mso-fareast-font-family:$bodyfont;
         mso-bidi-font-family:$bodyfont;
         mso-ansi-language:EN-GB;}
-p.AdmonitionTitle
+p.AdmonitionTitle, p.RecommendationTitle
```
[the remark this answers is not in the document]
Comment (ronaldtse) [reply lib/isodoc/ogc/html/ogc.scss]: @opoudjis this recommendation is right, let's fix it.

File: lib/isodoc/ogc/html/htmlstyle.scss [hunk 2]
```diff
@@ -420,6 +420,18 @@ p.Biblio, p.NormRef {
   }
 }
 
+.recommend {
```
[the remark this answers is not in the document]
Comment (ronaldtse) [reply lib/isodoc/ogc/html/htmlstyle.scss]: @opoudjis this recommendation is right, let's fix it.

**Comment by ronaldtse** [comment]
@opoudjis selectors should follow Hound recommendations like:
```
Name of mixin recommendationBlock should be written in all lowercase letters with hyphens instead of underscores
```

Unless these are old ones that are difficult to change, for now.

**Comment by opoudjis** [comment]
>> Name of mixin recommendationBlock should be written in all lowercase letters with hyphens instead of underscores

> @opoudjis selectors should follow Hound recommendations like:
> Unless these are old ones that are difficult to change, for now.

They are brand new with @strogonoff , but:

I refuse to implement anything Hound says, until someone reconciles Hound and Rubocop: their rules are contradictory. And that someone is not me.

I am going to insist on this from now on. I will not waste time on fixing things in Hound that Rubocop may contradict.

**Comment by strogonoff** [comment]
I don’t appreciate default Hound’s SCSS requirements. First, dashes interfere with the autocomplete system I use; second, I think it’s better to use native CSS style (kebab-case) to native CSS features and differentiate it with camelCase for SASS features. But if necessary I can adjust.

+1 to rule consistency!

**Comment by strogonoff** [comment]
Apparently (?), SASS allows to use dashes and underscores interchangeably in imported mixins; no such issue with camelCase. Not a big deal though.

Unlike e.g. Python, there is no strong convention across the industry so I went with my preference, but I have no strong opinion on this.

I raised a question about following Hound guidelines before but got the impression it’s not a priority yet.

Note that some of the existing SCSS already violated the rules before the refactor, so while I’m open to reformatting the code to conform I’d be inclined to do a style consistency cleanup as a bulk change outside of the refactor effort.

**Comment by opoudjis** [comment]
This will never be a priority, so it needs to be artificially made a priority. @ronaldtse, is Sebastian around? He was maintaining our rubocop when I'd last checked a year ago.

**Comment by ronaldtse** [comment]
Ping @skalee can you have a look at our Rubocop?

**Comment by skalee** [comment]
Yes, I'll take a look at it tomorrow. Having said that, Rubocop is a Ruby linter only, and it has nothing to do with SASS. Hound CI surely integrates other tools for that. I'll investigate how to configure them. Regarding Rubocop, I'll consider updating our style guides as well. We have frozen Rubocop version in Hound CI to avoid constant issues with breaking changes, but let's update them once for a while.

**Comment by skalee** [comment]
BTW I see that there are false positives in `lib/isodoc/ogc/reqt.rb` which clearly indicate that Hound is not using our style guide for this project:

> Lint/AmbiguousOperator: Ambiguous keyword splat operator. Parenthesize the method arguments if it's surely a keyword splat operator, or add a whitespace to the right of the ** if it should be a exponent.
> Style/StringLiterals: Prefer single-quoted strings when you don't need string interpolation or special symbols.
> Layout/ExtraSpacing: Unnecessary spacing detected.

I'll address that as well.

**Pull request merged**